examples/todomvc/src/store/index.js

```diff
@@ -0,0 +1,28 @@
+import { createStore, simpleObject, keyValue } from 'k-simple-state'
+import listeners from './listeners'
+
+export default createStore(
+  {
+    data: {
+      todos: {
+        all: { type: 'keyValue', key: 'id' },
```

Comment:
use `keyValue` function here

Reply:
I do it on purpose since TODOMVC is a show case